apiextensions/storageversion/cmd/migrate/main.go

```diff
@@ -31,6 +33,10 @@ import (
 	"knative.dev/pkg/signals"
 )
 
+const (
+	IgnoreNotFound = "MIGRATE_CRD_IF_FOUND_NO_FAILURE"
```

Comment:
This name is really confusing and not obvious

```suggestion
	IgnoreNotFoundEnvKey = "IGNORE_NOT_FOUND"
```